src/com/hms/flexyosisoftconnector/Main.java

```diff
@@ -21,129 +26,106 @@
   /** Application Minor Version Number */
   static final int MINOR_VERSION = 0;
 
-  /** Minimum amount of free memory before data trimming occurs */
-  static final long MINIMUM_MEMORY = 5000000;
-
-  /** Number of tags to trim when memory is low */
-  static final int TAGS_TO_TRIM = 2;
-
   /** Default name of a new Ewon */
   static final String DEFAULT_EWON_NAME = "eWON";
 
   /** Filename of connector config file */
   static final String CONNECTOR_CONFIG_FILENAME = "/usr/ConnectorConfig.json";
 
-  /** Configuration of PI connector */
-  static OSIsoftConfig piConfig;
-

```

Comment:
Are these changes really related to using the new queue? Seems like there is more going on in this commit